appdata.patch

```diff
@@ -0,0 +1,25 @@
+From 815c21eec86d2a35ecd4f1d7d7376389b17a284b Mon Sep 17 00:00:00 2001
+From: bbhtt <bbhtt.zn0i8@slmail.me>
+Date: Tue, 18 Jul 2023 11:01:06 +0530
+Subject: [PATCH] Fix screenshot location
+
+---
+ share/metainfo/com.mousepawmedia.omission.appdata.xml | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/share/metainfo/com.mousepawmedia.omission.appdata.xml b/share/metainfo/com.mousepawmedia.omission.appdata.xml
+index 27cb7d7..f1e576f 100644
+--- a/share/metainfo/com.mousepawmedia.omission.appdata.xml
++++ b/share/metainfo/com.mousepawmedia.omission.appdata.xml
+@@ -40,7 +40,7 @@
+     <screenshots>
+         <screenshot type="default">
+             <caption>Omission Gameplay: Timed Mode</caption>
+-            <image>https://mousepawgames.com/img/screenshots/omission/omission_timed.jpg</image>
++            <image>https://raw.githubusercontent.com/bbhtt/com.mousepawmedia.omission/runtime-44/com.mousepawmedia.omission.png</image>
```

Comment:
@CodeMouse92 You should update this URL to upstream repo (copy the screenshot from here) or https://github.com/flathub/com.mousepawmedia.omission/blob/master/com.mousepawmedia.omission.png

Otherwise, the next build will fail as it points to my branch.

Reply:
The bad URL was because I forgot to renew the domain. Derp! That should be fixed now.